pkg/cmdutil/userio/text_input.go

```diff
@@ -111,3 +112,14 @@ func (m textInputModel[V]) View() string {
 	}
 	return s.String()
 }
+
+func MaxLines(out strings.Builder) int {
```

Comment:
Not sure if the input parameter should be named `out`. Maybe we can give it a better name?